src/Angor/Client/Pages/Invest.razor

```diff
@@ -614,6 +547,7 @@ else
     private string nsecValue = "";
     private string npubValue = "";
     private string founderNpub = "";
+    private string currentUserPrivateKeyHex = "";
```

Comment:
This should not be kept as a long term parameter on the page

Reply:
probably what you mean is that we should null it when we close the component?